src/Models/IncompressibleModels/set_incompressible_model.jl

```diff
@@ -1,3 +1,6 @@
+using Oceananigans.TimeSteppers: update_state!,
+using Oceananigans.TimeSteppers: calculate_pressure_correction!, pressure_correct_velocities!
```

Comment:
```suggestion
using Oceananigans.TimeSteppers: update_state!, calculate_pressure_correction!, pressure_correct_velocities!
```

I think this should fix the failing tests.